ReduxTestTask/Classes/NewsList/View/NewsListController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  NewsListController.swift
+//  ReduxTestTask
+//
+//  Created by Polina Hill on 10/1/19.
+//  Copyright © 2019 Polina Hill. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+
+final class NewsListController: UIViewController {
+
```

Comment:
Better to call it NewsListViewController or NewsListVC, because we have IGListKit controllers.